src/SW_Site.c

```diff
@@ -630,6 +631,7 @@ double SW_swcBulk_minimum(
 				"`SW_swcBulk_minimum()`: SWRC (type %d) is not implemented.",
 				swrc_type
 			);
+            return theta_min_theoretical; // Exit function prematurely due to error

```

Comment:
Let's not return values that could be confused with real values on error
```
LogError(
				LogInfo,
				LOGERROR,
				"`SW_swcBulk_minimum()`: SWRC (type %d) is not implemented.",
				swrc_type
			);
            return theta_min_theoretical; // Exit function prematurely due to error
```
Let's return `SW_MISSING` (as done elsewhere)